arm-iothub/2016-02-03/swagger/iothub.json

```diff
@@ -1433,15 +1413,15 @@
       }
     },
     "ErrorDetails": {
-      "description": "The properties related to the details of an error.",
+      "description": "Error details.",
```

Comment:
For 1. The pascal case is correct. 2. Added required properties for responses which I had purposefully avoided earlier as it didn't make much sense.